dsc/workflows/sccs.py

```diff
@@ -10,4 +10,15 @@ class SCCS(SimpleCSV):
     """
 
     workflow_name: str = "sccs"
-    metadata_mapping_path: str = "dsc/workflows/metadata_mapping/sccs.json"
+
+    @property
+    def metadata_mapping_path(self) -> str:
+        return "dsc/workflows/metadata_mapping/sccs.json"
+
+    @property
+    def s3_bucket(self) -> str:
+        return "awaiting AWS infrastructure"
+
+    @property
+    def output_queue(self) -> str:
+        return "awaiting AWS infrastructure"
```

Comment:
[Non-blocking] Since we are planning to create an output queue for every workflow, this might resemble something like `dss-sccs-output`, right?

Reply:
You are very likely correct but I think we'll need a PR for updating these values once the AWS infra is done so I'd rather not guess until they've been created